test/node/plugins.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var $nocca = require('../../index');
+
+var Nocca = new $nocca({
+    endpoints: {
+        'google': {
+            targetBaseUrl: 'http://www.google.com/'
+        },
+        'yahoo': {
+            targetBaseUrl: 'http://www.yahoo.com/',
+            keyGenerator: 'keyGeneratorFactory',
+            keyGeneratorFactoryMethod: 'urlAndHeadersKeyGeneratorBuilder'
+        },
+        '_default': {
+            targetBaseUrl: 'http://localhost:3004/'
+        }
+    },
+    keyGenerator: 'cherryPickingKeygen',
```

Comment:
So the default keyGenerator here is the cherryPickingKeygen? Looking at its features, it pretty much replaces the keyGeneratorFactory :P